e2e/schematics/affected.test.ts

```diff
@@ -184,6 +184,11 @@ describe('Affected', () => {
       expect(linting).toContain('Running lint for myapp-e2e');
       expect(linting).toContain('Running lint for mypublishablelib');
 
+      const lintWithJsonFormating = runCommand(
+        'npm run affected:lint -- --files="libs/mylib/src/index.ts" -- --format json'
+      );
+      expect(lintWithJsonFormating).toContain('[]');
```

Comment:
how does the same command look now?

Reply:
It will stdout the JSON return.. (or whichever format they choose). I'm not sure how useful it is in practice, but it tests the argument parsing... Let me add a check for the flags being present.